src/downloader/download.py

```diff
@@ -37,11 +38,8 @@ def download_org_workflows_and_actions() -> None:
 
     We are trying to cache the downloads as much as we can to reduce redundant download attempts.
     """
-    log.debug("[+] Starting organization repository iterator")
-
-    for organization in Config.org_name:
-        log.debug(f"[+] Scanning {organization}")
-        generator = get_organization_repository_generator(organization)
+    for account in Config.account_name:
+        generator = get_account_generator(account)
```

Comment:
How about adding a debug massage: 
log.debug(f"[+] Scanning {account}")

Reply:
There is a log inside the `get_account_generator` function:

``` python
if account_type == "User":
        log.info(f"[+] Scanning user: {account_name}")
        return get_user_repository_generator(account_name)

    elif account_type == "Organization":
        log.info(f"[+] Scanning organization: {account_name}")
        return get_organization_repository_generator(account_name)
```